www/havm10/SeminarPage02/css/main.css

```diff
@@ -0,0 +1,188 @@
+* {
+    box-sizing: border-box;
+}
+
+.MapPart {
+    display: flex;
+    width: 100%;
+}
+
+.content {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 100%;
+    margin: 10px;
+}
+
+.save-content-conteiner {
+    max-width: 25%;
+    padding-left: 5px;
+    padding-right: 20px;
+    min-width: 25%;
+    border-width: 2px; 
+    border-style: solid; 
+    border-radius: 10px;
+}
+
+.markers {
+    display: flex; 
+    flex-wrap: wrap;
+    width: 40%;
+    border-width: 2px; 
+    border-style: solid; 
+    margin-left: 10px;
+    margin-right: 10px;
+    border-radius: 10px;
+}
+
+.sections {
+    display: flex;
+    flex-wrap: wrap;
+    min-width: 100%;
+    margin: 10px;
+}
+
+.section {
+    min-width: 50%;
+}
+
+
+.output {
+    display: flex;
+    width: 100%;
+    width: 40%;
+    padding-left: 5px;
+    padding-right: 5px;
+    border-width: 2px; 
+    border-style: solid; 
+    border-radius: 10px;
+}
+
+.data{
+    min-width: 60%;
+}
+
+#canvas-draw {
+    padding: 10px;
+    height: 300px;
+    border: 1px;
+    border-style: solid;
+    width: 35%;
+    border-width: 2px; 
```

Comment:
ty bordery se mi moc nelibi, vypadaji prilis staromodne

Reply:
odstraněno, vzhled jsem trochu upravil, předělal jsem rozložení layoutu. Podle mého názoru je to nyní přehlednější